src/main/java/com/robertx22/age_of_exile/aoe_data/database/spells/schools/PetSpells.java

```diff
@@ -28,6 +28,7 @@ public void registerAll() {
                 )
                 .weaponReq(CastingWeapon.ANY_WEAPON)
                 .onHit(PartBuilder.damage(SpellCalcs.PET_BASIC, Elements.Physical))
+                .isBasicAttack()
```

Comment:
May you explain me how do I test that datapack entry "is_basic_attack" would work?